src/state/reducers/month-state/employee-info.reducer.ts

```diff
@@ -14,36 +13,6 @@ import {
   ScheduleActionType,
 } from "./schedule-data/schedule.actions";
 
-function fromFractionToHours(fraction: string): number {
```

Comment:
To już nie jest używane, bo teraz dispatchuję do stanu etat pracownika jako liczbę zmiennoprzecinkową